src/common/common.hpp

```diff
@@ -104,8 +104,19 @@ static inline bool VectorContainsString(const vector<string> &vec, const string
 	return std::find(vec.begin(), vec.end(), str) != vec.end();
 }
 
+static inline string StringVectorToString(const vector<string> &vec, const string delim = ",") {
+	string ret = "{";
+	auto sz = vec.size();
+	for (decltype(sz) i = 0; i < (sz - 1); i++) {
+		ret += "\"" + vec[i] + "\"" + delim;
+	}
+	ret += "\"" + vec[sz - 1] + "\"" + "}";
```

Comment:
This is almost the same as `JoinStrings` above.

Reply:
Yes, but `JoinStrings()` doesn't add the quotes and curly brackets. I could use it here and add the curly brackets, but I think the quotes are also nice.